python/tvm/relax/distributed/struct_info.py

```diff
@@ -0,0 +1,70 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=redefined-builtin, invalid-name
+"""Struct Info for distributed tensor."""
+import tvm
+from tvm.relax.struct_info import StructInfo, TensorStructInfo
+from tvm.ir import Span
+from tvm.runtime.object import Object
+
+from .global_info import DeviceMesh
+from . import _ffi_api
+
+
+@tvm._ffi.register_object("relax.distributed.Placement")
+class Placement(Object):
+    """Describes how data is distributed in each dimension of the device mesh
+
+    Parameters
+    ----------
+    text_format: str
```

Comment:
Can we explicitly construct placement not from text format?

I think that would be the case, in this case use a normal constructor here, and use create_placement(text_repr)